### DISCUSSION TITLE
Unity 2022.3.LTS Support?

### DISCUSSION THREAD

**Discussion by RMichaelPickering** [Voltstro-Studios/UnityWebBrowser#247] [Requests]
We're near the end of 2023 already, and Unity has moved the 2022 stream to LTS status! Let's just assume there is still community love left for Unity3D in 2024, what is the plan for supporting UWB on its latest LTS stream please? Are there any known gotchas? Would help in testing on Unity 2022.3 be needed?

**Comment by Voltstro** [comment]
Not aware of any issues using UWB on Unity 2022+, no one has ever reported any Unity version specific issues.

**Comment by RMichaelPickering** [comment]
I did some testing in 2023.2 and other than one script in sample not working, and UI not responding (presumably due to input not being correctly set up, possibly related also to the non-working script), I found no issues.
